birdhouse/deployment/deploy-data-specific-image

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+# Deploy data from git repo(s) to local folder(s).
+# This script is run directly on a specific image (such as pavics/crim-jupyter-eo or pavics/crim-jupyter-nlp).
+# It will be used to download and update the different tutorial notebooks associated with a specific image.
+#
+# This is meant to be run on the same host running PAVICS.
+#
+# The data details is specified in a yaml config file (TEMPLATE_CONFIG_YML), using this format :
+# - repo_url: path to the github repo (ex.: https://github.com/crim-ca/pavics-jupyter-images)
+#   branch: name of the branch containing the required version of the data
+#   source_path: path of the desired file or folder in the source repo
+#   dest_dir: directory where the data will be copied to in the specific image where the script is run
```

Comment:
I don't think it should be an option for the image to choose. The host is mounting the directory $JUPYTERHUB_USER_DATA_DIR and by convention the dest_dir should be $JUPYTERHUB_USER_DATA_DIR/tutorial-notebooks/<image_name>

In fact, the host volume to be mounted should be $JUPYTERHUB_USER_DATA_DIR/tutorial-notebooks/<image_name> and not $JUPYTERHUB_USER_DATA_DIR. The image can only see it's own folder. An env var holding that folder, something like NOTEBOOK_DEST_DIR would also be set so that the script knows where to put files.

Reply:
I agree that this could be a good option.

Also see my other comment : https://github.com/bird-house/birdhouse-deploy/pull/150#discussion_r616978832